pom.xml

```diff
@@ -382,52 +382,21 @@
             <artifactId>oauth2-oidc-sdk</artifactId>
             <version>9.9.1</version>
         </dependency>
-        <!-- EXPERIMENTAL: -->
-        <!-- lyncode xoai OAI-PMH implementation: -->
-        <!-- unfortunately, their 4.10 version -->
-        <!-- is still buggy. As an experiment, I'm using -->
-        <!-- a patched version I built locally. -->
-        <!-- (pull requests pending - L.A. -->
-        <!-- These locally-built XOAI libraries are still a serious debt -->
-        <!-- that needs to be addressed. Adding explicit "exclusion" entries -->
-        <!-- below for the immediate purpose of getting rid of the log4j library -->
-        <!-- that was being added to the project via these dependencies. (L.A. - Jan. 2022) -->
-        <!-- (note the reload4j explicitly added below as a replacement! -->
-        <dependency>
-            <groupId>com.lyncode</groupId>
+        <!-- New and Improved GDCC XOAI library! --> 
+        <dependency>
+            <groupId>io.gdcc</groupId>
             <artifactId>xoai-common</artifactId>
-            <version>4.1.0-header-patch</version>
+            <version>${gdcc.xoai.version}</version>
         </dependency>
         <dependency>
-            <groupId>com.lyncode</groupId>
+            <groupId>io.gdcc</groupId>
             <artifactId>xoai-data-provider</artifactId>
-            <version>4.1.0-header-patch</version>
-            <exclusions>
-                <exclusion>
-                    <groupId>log4j</groupId>
-                    <artifactId>log4j</artifactId>
-                </exclusion>
-            </exclusions>
+            <version>${gdcc.xoai.version}</version>
```

Comment:
Would it make sense to release a BOM for this lib and use it in `<dependencyManagement>` to avoid these repeats here? (Following [JLBP-4](http://java.jonathangiles.net/JLBP-4))

Reply:
This one I'm not sure about honestly, if it's worth bothering.